### PR TITLE
005.txt and 006.txt

### PR DESCRIPTION
**Pull request by Antielfi**
This conclude the prologue part in theory. Unfortunally I can't test ……it right now but hopefully is ok

### CODE REVIEW

**Comment by Antielfi** [comment]
Aaaaah soooo fuuu, well there are more file now e.e

File: Cornelica/scenario/007_01.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@
 @bs f=アル@普通 body=-1 face=0 x=-350 y=50 t=200
 @bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
 
-[Lily] "The food is also delicious, so many people come for food too"
+[Lily]"The food is also delicious, so many people come for to eat too"
```
Comment (TwinDragon) [inline Cornelica/scenario/007_01.txt]: `come for to eat too`? - Grammar-wise, this make no sense. At least to me. Maybe just dropping `for` might work.

File: Cornelica/scenario/007_07.txt [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 @bs f=リリィ@困惑 op=困惑 x=350 y=50 t=200
 
-[Lily] "This place ... I'm not very fond of this place ... so please..."
+[Lily]"This place ...I'm not very fun of this place...\nso please..."
```
Comment (TwinDragon) [inline Cornelica/scenario/007_07.txt]: `I'm not very fun` doesn't make sense. XD Which is why I think "fond" was more suited I believe.

File: Cornelica/scenario/010.txt [hunk 3]
```diff
@@ -182,45 +182,45 @@
 
 @huki b=7 id=12
 @bs f=リリィ@困惑 op=困惑 body=-1 face=0 x=350 y=50 t=200
-[Lily] "... After all ... ... when I was hugged a little while ago ...."
+[Lily]"...After all...when you was hugged a little while ago...."
```
Comment (TwinDragon) [inline Cornelica/scenario/010.txt]: I think `when you were hugged` makes more sense here.

File: Cornelica/scenario/010.txt [hunk 4]
```diff
@@ -182,45 +182,45 @@
...
 
 @huki id=11 b=2
 @bs f=アル@困り_頬 op=ハテナ body=-1 face=0 x=-350 y=50 t=200
 @wait t=600
 
 @bs f=リリィ@焦り body=-1 face=0 x=350 y=50 t=200
 
-[Lily] "Uh ... ... us, Succubi are always exuding pheromones \ C [0] that is \ C [2]" charm ""
+[Lily]"Uh... us, Succubi have magical power called \C[2]charm\C[2]"
```
Comment (TwinDragon) [inline Cornelica/scenario/010.txt]: That last `\C[2]` probably should be a 0 instead.

File: Cornelica/scenario/010.txt [hunk 5]
```diff
@@ -182,45 +182,45 @@
...
 @bs f=アル@焦り_頬 op=焦り body=-1 face=0 x=-350 y=50 t=200
 @bs f=リリィ@困惑 body=-1 face=0 x=350 y=50 t=200
 
-[Lily] "I have nearly no magical power on my own, so it's okay to keep it down to no effect on Al, but ..."
+[Lily]"I have nearly no magical power on my own, so it's okay to keep it down to no effect on you, but..."
 
 @bs f=リリィ@蔑み body=-1 face=0 x=350 y=50 t=200
 
-[Lily] "Because I do not purposely suppress my power, \ C [2]" Charm 's magical power "will accumulate in my body \ C [0]"
+[Lily]"Because other succubus don't purposely suppress thier \npower, \C[2]charm will accumulate into your body \C[0]"
```
Comment (TwinDragon) [inline Cornelica/scenario/010.txt]: Just a spelling mistake: `their`

File: Cornelica/scenario/007_01.txt (hunk 1 above)
Comment (Antielfi) [reply Cornelica/scenario/007_01.txt]: Copy-pasta madness, probably i was daydreaming @.@
Yeah... it's sound better

File: Cornelica/scenario/007_07.txt (hunk 2 above)
Comment (Antielfi) [reply Cornelica/scenario/007_07.txt]: yeah... still mindless copy error :D

File: Cornelica/scenario/010.txt (hunk 3 above)
Comment (Antielfi) [reply Cornelica/scenario/010.txt]: If my old prof see this.... :D that's why is better to doublecheck my english

File: Cornelica/scenario/010.txt (hunk 4 above)
Comment (Antielfi) [reply Cornelica/scenario/010.txt]: Yep, it's important too... arrgg....

**Comment by Antielfi** [comment]
I will fix the error in a bit, and after that i will try to find the other big problem in the free time between work

File: Cornelica/scenario/007_06.txt [hunk 6]
```diff
@@ -10,6 +10,16 @@
...
+[Lily]"This is the brothel..."
+
+@bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
+
+[Lily]"... or, where it was..."
+
+@bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
+
+[Lily]"but the male escorts quickly became impotent... \n...so it went bankrupt."
+
```
Comment (TwinDragon) [inline Cornelica/scenario/007_06.txt]: o.O - Are these added by mistake or something? They're the same as right below these lines.

Comment (Antielfi) [reply Cornelica/scenario/007_06.txt]: grrr.... that file is screw. I can't figure it out. Right before those is had like
<<<<<<<<<<HEAD
stuff... i will try to re-fixit, but if keep fucking up i can send you the correct file so you can fix directly into the code

**Pull request merged**